app/models/street_cafe.rb

```diff
@@ -1,5 +1,15 @@
 class StreetCafe < ApplicationRecord
-  def self.get_ls2_chair_list
-    where("post_code LIKE 'LS2%'").order(:number_of_chairs).pluck(:number_of_chairs)
+  class << self
+    def get_ls2_chair_list
+      where("post_code LIKE 'LS2%'").order(:number_of_chairs).pluck(:number_of_chairs)
+    end
+
+    def cafes_by_category(*args)
```

Comment:
Went with args to reduce very similar methods. Originally had it as StreetCafe.small_cafes